my_list.rb

```diff
@@ -0,0 +1,19 @@
+# Create a class MyList that has an instance variable @list.
+
+require_relative 'my_enumerable'
+
+class MyList
+  def initialize(*list)
+    @list = list
```

Comment:
- [ ] You need to add these codes in order to make the line 17 runs as excepected 100%, just for safety
```suggestion
    @list = []
        list.each { |arg| @list << arg }
```